common/models.py

```diff
@@ -108,3 +108,8 @@ def save(self, *args, **kwargs):
             # reason. Should not be needed in normal usage.
             self.person_deleted_at = None
         return super().save(*args, **kwargs)
+
+
+class SQCount(models.Subquery):
```

Comment:
Renaming `SQCount` to e.g. `SubqueryCount` would make the code more readable and more easily understandable IMHO. I base this on Zen of Python's "Explicit is better than implicit." and "Readability counts.".

Reply:
Taken into account